django-stubs/test/runner.pyi

```diff
@@ -83,16 +92,30 @@ class ParallelTestSuite(TestSuite):
     initial_settings: Any
     serialized_contents: Any
     def __init__(
-        self, subsuites: list[TestSuite], processes: int, failfast: bool = ..., buffer: bool = ...
+        self,
+        subsuites: list[TestSuite],
+        processes: int,
+        failfast: bool = ...,
+        debug_mode: bool = ...,
+        buffer: bool = ...,
     ) -> None: ...
     def run(self, result: Any) -> Any: ...  # type: ignore[override]
+    def initialize_suite(self) -> None: ...
+
+class Shuffler:
+    hash_algorithm: ClassVar[str]
+
+    def __init__(self, seed: int | None = ...) -> None: ...
+    @property
+    def seed_display(self) -> str: ...
+    def shuffle(self, items: list[TestCase], key: Callable[[TestCase], str]) -> list[TestCase]: ...
 
 class DiscoverRunner:
     test_suite: type[TestSuite]
     parallel_test_suite: type[ParallelTestSuite]
     test_runner: type[TextTestRunner]
     test_loader: TestLoader
-    reorder_by: tuple[SimpleTestCase, ...]
+    reorder_by: tuple[type[TestCase], type[SimpleTestCase]]
```

Comment:
I think that it should be `tuple[type[X], ...]`, where `X` is a proper type name.

Why? Because it would be easier to subclass and change. Or maybe even `Iterable` / `Sequence`.